tests/phpstan/phpstan-1.7.8.neon

```diff
@@ -1,2 +1,11 @@
 includes:
 	- %currentWorkingDirectory%/tests/phpstan/phpstan.neon
+
+parameters:
+	ignoreErrors:
+		## Compatibility with PrestaShop 8.0
+		- '#^Call to method getInstanceByName\(\) on an unknown class PrestaShop\\PrestaShop\\Core\\Module\\ModuleRepository.#'
```

Comment:
This method doesn't exists in PrestaShop 8.0, I don't think it should be ignored.

Also, this is the phpstan config for PrestaShop 1.7.8